lib/desktop_screens/desktop_welcome_screen/widgets/welcome_screen_home.dart

```diff
@@ -222,6 +222,11 @@ class _WelcomeScreenHomeState extends State<WelcomeScreenHome> {
                                       : CommonButton(
                                           isFileShareFailed ? 'Resend' : 'Send',
                                           () async {
+                                            await Provider.of<MyFilesProvider>(
+                                                    NavService
+                                                        .navKey.currentContext!,
+                                                    listen: false)
+                                                .deleteMyfilekeys();
```

Comment:
Why are we calling deleteMyfilekeys function in send button?
This was removed as part of the fix that was not allowing us to send files in desktop app.

Reply:
Sorry I added it accidentally while resolving conflicts.
Removed now.